ui/pages/confirm-transaction/confirm-token-transaction-switch.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { Switch, Route } from 'react-router-dom';
+import {
+  CONFIRM_APPROVE_PATH,
+  CONFIRM_SEND_TOKEN_PATH,
+  CONFIRM_TRANSACTION_ROUTE,
+  CONFIRM_TRANSFER_FROM_PATH,
+} from '../../helpers/constants/routes';
+import { transactionFeeSelector } from '../../selectors';
+import ConfirmApprove from '../confirm-approve';
+import ConfirmSendToken from '../confirm-send-token';
+import ConfirmTokenTransactionBase from '../confirm-token-transaction-base';
+import ConfirmTransactionSwitch from '../confirm-transaction-switch';
+
+const { useAssetDetails } = require('../../hooks/useAssetDetails');
+
+export const ConfirmTokenTransactionSwitch = ({ transaction }) => {
```

Comment:
Why is this a named export, what do you think about using a default export here?

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/13788/commits/6fb5f7e725635b5f364fe4ec7d0d13024284b59a